docs/resources/ignore_rule.md

```diff
@@ -134,7 +134,10 @@ Optional:
 
 ## Import
 
-Ignore rules can be imported using their IDs, e.g.
+Import is supported using the following syntax:
+
+```shell
+terraform import xray_ignore_rule.my-rule rule-name
```

Comment:
Do you think it's necessary? The import statement example was already there. Also, it's in the template. The next time doc will be generated, it will be removed.